src/ipc/CMakeLists.txt

```diff
@@ -2,6 +2,47 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or https://opensource.org/license/mit/.
 
+# Build local libmultiprocess/ directory, unless an external libmultiprocess
+# package is being used instead.
+if (NOT WITH_EXTERNAL_LIBMULTIPROCESS)
+  # Unconditionally set BUILD_TESTING to ON. BUILD_TESTING is a standard cmake
+  # variable that controls whether CTest is used. In the bitcoin build, CTest is
+  # always used and we do not want an option to turn it off. But in
+  # libmultiprocess, CTest is optional and BUILD_TESTING needs to be set in
+  # order to add libmultiprocess unit tests to the CTest configuration.
+  set(BUILD_TESTING ON)
```

Comment:
@hebasto should we unset `CMAKE_RUNTIME_OUTPUT_DIRECTORY` here to avoid `mpgen` and `mpgen` landing in the `bin/` directory? Maybe it would be better to move the `libmultiprocess` subtree up one level?

Reply:
> @hebasto should we unset `CMAKE_RUNTIME_OUTPUT_DIRECTORY` here to avoid `mpgen` and `mpgen` landing in the `bin/` directory? Maybe it would be better to move the `libmultiprocess` subtree up one level?

I like the idea of moving the add_subdirectory call up a level so to avoid this problem so will experiment with this.